mail/2.0/_index.md

```diff
@@ -33,19 +33,21 @@ Please reference that ballot for the official results.
 
 ## Release Review
 
-The Specification Committee Ballot concluded successfully on 2020-mm-dd with the following results.
-
-| Representative                                 | Representative for: | Vote |
-|------------------------------------------------|---------------------|------|
-| Kenji Kazumura, Michael DeNicola               | Fujitsu             |      |
-| Dan Bandera, Kevin Sutter                      | IBM                 |      |
-| Bill Shannon, Ed Bratt                         | Oracle              |      |
-| Mark Wareham, Steve Millidge                   | Payara              |      |
-| Scott Stark, Mark Little                       | Red Hat             |      |
-| David Blevins, Cesar Hernandez                 | Tomitribe           |      |
-| Ivar Grimstad                                  | EE4J PMC            |      |
-| Alex Theedom                                   | Participant Members |      |
-| Werner Keil                                    | Committer Members   |      |
-|                                                | Total               |      |
+The Specification Committee Ballot concluded successfully on 2020-09-11 with the following results.
+
+| Representative                                      | Representative for: | Vote |
+|-----------------------------------------------------|---------------------|------|
+| Kenji Kazumura                                      | Fujitsu             |  +1  |
+| Dan Bandera, Kevin Sutter                           | IBM                 |  +1  |
+| Ed Bratt, Dmitry Kornilov                           | Oracle              |  +1  |
+| Andrew Pielage, Matt Gill                           | Payara              |  +1  |
+| Scott Stark, Mark Little                            | Red Hat             |  +1  |
+| David Blevins, Jean-Louis Monteiro, Cesar Hernandez | Tomitribe           |  +1  |
+| Ivar Grimstad                                       | EE4J PMC            |  +1  |
+| Marcelo Ancelmo, Martijn Verburg                    | Participant Members |  +1  |
+| Werner Keil                                         | Committer Members   |  +1  |
+| Scott (Congquan) Wang                               | Enterprise Members  |  +1  |
+|                                                     | Total               |  10  |
 
 The ballot was run in the [jakarta.ee-spec mailing list]()
+
```

Comment:
You need to put the url for the BALLOT starting message so that someone could track the actual votes, if required.